ddl/ddl_worker.go

```diff
@@ -1377,9 +1377,13 @@ func updateSchemaVersion(d *ddlCtx, t *meta.Meta, job *model.Job, multiInfos ...
 		if err != nil {
 			return 0, errors.Trace(err)
 		}
-		affects := make([]*model.AffectedOption, len(newSchemaIDs))
+		affects := make([]*model.AffectedOption, len(newSchemaIDs)-1)
 		for i, newSchemaID := range newSchemaIDs {
-			affects[i] = &model.AffectedOption{
+			// Do not add the first table to AffectedOpts. Related issue tidb#47064.
+			if i == 0 {
```

Comment:
Could we not change these codes and remove line1393-1295 to fix it?

Reply:
I think it can be fixed, but it does have potential r risks. We may need to remove "applyTableUpdate" like "applyCreateTables" for safety.

Or we can do this according to your PR first, and then do unification.

Reply:
I think we can `do unification` later and i'm pleased to do that. We can choose the second method fisrt, because it is a simple and reliable way to fix the problem.

Reply:
Yes. We can fix the bug first. 
Thank you very much for the follow-up help to unify this logic.